src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../common/variables';
+
+.bcs-ActivityThread {
+    .bcs-ActivityThread-button {
+        margin-left: $sidebarActivityFeedSpacingHorizontal;
+        color: $bdl-box-blue;
+        font-weight: bold;
+
+        &:hover {
+            margin-left: $sidebarActivityFeedSpacingHorizontal;
+            font-weight: bold;
```

Comment:
Do these need to be duplicated to override other button styles on hover?

Reply:
unfortunately yes,  because of the styling for `.btn-plain:hover`